### :woman_technologist: Hi, I'm Selin, a computational biologist and post-doctoral scientist

- I'm a post-doc based jointly in [Dr. Anshul Kundaje's lab](https://kundajelab.stanford.edu) and [Dr. Will Greenleaf's lab](https://greenleaf.stanford.edu) in the Department of Genetics at Stanford University
- I did my PhD in [Quantitative Life Sciences](https://www.mcgill.ca/qls/) in [Dr. Claudia Kleinman's lab](https://functionalgenomics.ca/) at McGill University, where I used integrative computational analyses of omics data types to answer biological questions about cell identity, cell fate, and gene regulation in brain development and  brain cancer
- :dna: I care about bioinformatics/computational biology research that emphasizes [reproducibility](https://github.com/sjessa/rr), [transparency](https://github.com/fungenomics/G34-gliomas), and [reusability](https://zenodo.org/record/6929428#.YwzUIi8r1pR)
- :book: Find my papers on [Google Scholar](https://scholar.google.ca/citations?user=rkcx2NwAAAAJ&hl=en) and find me on [LinkedIn](https://www.linkedin.com/in/selinjessa/)

__Here are the key research projects I worked on during my PhD:__

### A single-cell RNAseq atlas for the developing mouse forebrain and brainstem 

We assembled a scRNAseq reference dataset of >100k cells from nine timepoints in the developing mouse brain to profile
cell types and their expression patterns and dynamics.

- [Stalled developmental programs at the root of pediatric brain tumors](https://doi.org/10.1038/s41588-019-0531-7), Jessa et al, Nature Genetics, 2019
- :package: [`cytobox`](https://fungenomics.github.io/cytobox/), R package for scRNAseq data visualization
- [`braindex`](http://cc-shiny-01.functionalgenomics.ca/braindex/clusters/), custom interactive data portal implemented in R Shiny
- [`braindex` source code](https://github.com/fungenomics/braindex)

### A data-driven approach to identify the cell-of-origin in pediatric brain tumors

Leveraging our brain reference atlas, we designed approaches to computationally integrate primary patient tumor omics data (scRNAseq, scATACseq, bulk RNAseq, ChIPseq, HiC) with 
the atlas in order to identify, for each subtype of pediatric brain tumors, their most likely cell-of-origin.

- [Stalled developmental programs at the root of pediatric brain tumors](https://doi.org/10.1038/s41588-019-0531-7), Jessa et al, Nature Genetics, 2019
- [Histone H3.3G34-mutant interneuron progenitors co-opt PDGFRA for gliomagenesis](https://www.cell.com/cell/fulltext/S0092-8674(20)31529-4), Chen*, Deshmukh*, Jessa* et al, Cell, 2020
  - [`G34-gliomas`](https://fungenomics.github.io/G34-gliomas/), the code & analysis for my contributions to the Chen et al study
- [K27M in canonical and noncanonical H3 variants occurs in distinct oligodendroglial cell lineages in brain midline gliomas](https://www.nature.com/articles/s41588-022-01205-w#Abs1), Jessa*, Mohammadnia*, Harutyunyan*, et al, Nature Genetics, 2022
  - [`HGG-oncohistones`](https://github.com/fungenomics/HGG-oncohistones), the the code & analysis for the H3K27M gliomas study
