### ðŸ‘‹ Hi, I'm Selin, a computational biologist wrapping up my PhD in early 2023

I used integrative computational analyses of omics data types to answer biological questions about cell fate and gene regulation in brain development and  brain cancer. I care about bioinformatics workflows that emphasize reproducibility and transparency. :dna: :microscope: :computer: :woman_technologist:

Find my papers on Google Scholar [here](https://scholar.google.ca/citations?user=rkcx2NwAAAAJ&hl=en).

__Here are my key research projects:__

### `chromswitch`, An R package to detect chromatin state switches from epigenomic data

chromswitch implements a flexible method to detect chromatin state switches between samples
in two biological conditions in a specific genomic region of interest given peaks or chromatin state calls from ChIP-seq data.

- Associated publication: [Jessa and Kleinman, Bioinformatics, 2019](https://academic.oup.com/bioinformatics/article/34/13/2286/4846890?login=false)
- :package: [Bioconductor package](https://www.bioconductor.org/packages/release/bioc/html/chromswitch.html)
- [`chromswitch` source code](https://github.com/sjessa/chromswitch)
- [Reproducible analysis](https://sjessa.github.io/chromswitch-analysis/) using R Markdown [![DOI](https://zenodo.org/badge/103428474.svg)](https://zenodo.org/badge/latestdoi/103428474)

### A single-cell RNAseq atlas for the developing mouse forebrain and brainstem 

We assembled a reference dataset of >100k cells from nine timepoints in the developing mouse brain to profile
cell types and their expression patterns and dynamics.

-  Associated publication: [Jessa et al, Nature Genetics, 2019](https://doi.org/10.1038/s41588-019-0531-7)
- :package: [`cytobox`](https://fungenomics.github.io/cytobox/), R package for scRNAseq data visualization
- [`braindex`](http://cc-shiny-01.functionalgenomics.ca/braindex/clusters/), custom interactive data portal implemented in R Shiny
- [`braindex` source code](https://github.com/fungenomics/braindex/tree/master/clusters)

### A data-driven approach to identify the cell-of-origin in pediatric brain tumors

Leveraging our brain reference atlas, we designed approaches to computationally integrate primary patient tumor NGS data with 
the atlas in order to identify, for each subtype of pediatric brain tumors, their most likely cell-of-origin.

- Associated publications:
  - [Jessa et al, Nature Genetics, 2019](https://doi.org/10.1038/s41588-019-0531-7)
  - [Chen*, Deshmukh*, Jessa* et al, Cell, 2020](https://www.cell.com/cell/fulltext/S0092-8674(20)31529-4)
  - Jessa et al, Nature Genetics (accepted) on high-grade gliomas
- [Reproducible analysis](https://fungenomics.github.io/G34-gliomas/) for my contributions to the Chen et al study
- [Reproducible analysis](https://github.com/fungenomics/HGG-oncohistones) for our forthcoming study on high-grade gliomas [![DOI](https://zenodo.org/badge/503518963.svg)](https://zenodo.org/badge/latestdoi/503518963)

<img alt="R" src="https://img.shields.io/badge/r-%23276DC3.svg?style=for-the-badge&logo=r&logoColor=white"/> <img alt="Markdown" src="https://img.shields.io/badge/markdown-%23000000.svg?style=for-the-badge&logo=markdown&logoColor=white"/> <img alt="Python" src="https://img.shields.io/badge/python-%2314354C.svg?style=for-the-badge&logo=python&logoColor=white"/> <img alt="Jupyter" src="https://img.shields.io/badge/Jupyter-%23F37626.svg?style=for-the-badge&logo=Jupyter&logoColor=white" /> <img alt="GitHub" src="https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white"/>
